src/Corax/IndexSearcher/IndexSearcher.cs

```diff
@@ -65,6 +65,8 @@ public sealed unsafe partial class IndexSearcher : IDisposable
     private Lookup<Int64LookupKey> _entryIdToLocation;
     public FieldsCache FieldCache;
 
+    public long MaxMemoizationSize = 128 * 1024 * 1024;
```

Comment:
rename to MaxMemoizationSizeInBytes?

Reply:
done in https://github.com/ravendb/ravendb/pull/17088/commits/cd05998cd6c2382d0835d0d73f7241f224d1d238